src/main/resources/db/migration/V1_1_0__first_migration.sql

```diff
@@ -0,0 +1,60 @@
+alter table if exists pharmacy_branch
```

Comment:
tu jest okej, natomiast po wgraniu flywaya możesz dropnąć bazę i ona się załaduje z tych skryptów.

Zakładając że dołączam do projektu i odpalę skrypt który sprząta mi bazę która w sumie pierwotnie jest czysta. Więc zbędny ten kod. Rozumiem że sprzątasz bazę u siebie, ale ja bym to dla porządku po prostu usunął. Te dropy, alter table itd są spoko jak mamy np. skrypt wersja 50 i chcemy posprzątać coś co zostało popsute kilka skryptów temu, aplikacja działa na środowiskach i nie możemy już dropnąć bazy, albo zmieniła się logika biznesowa aplikacji i chcemy coś zmodyfikować. 

Także dla tego skryptu po prostu bym to ukrócił do tworzenia tabel bez modyfikacji i sprzątania